web/src/App.js

```diff
@@ -423,6 +424,12 @@ class App extends Component {
             <AdminTab account={this.state.account} />
           </div>
         </Route>
+        <Route exact path="/admin/poster">
+              <div id={pcBrowser ? "Main" : ""}>
+                  {pcBrowser ? <div className="sep20" /> : null}
+                  <AdminPoster account={this.state.account} />
```

Comment:
It seems like that `account` is not used in `AdminPoster`. Can we remove it?

Reply:
> It seems like that `account` is not used in `AdminPoster`. Can we remove it?

yes,i think